errors/errors.go

```diff
@@ -4,6 +4,7 @@ import "errors"
 
 var (
 	ErrBadConfig               = errors.New("config: invalid config")
+	ErrCliBadConfig            = errors.New("cli-config: invalid config")
```

Comment:
the err msg should simply be "cli: bad config"

Reply:
ok, will be fixed.